crates/aptos/src/common/init.rs

```diff
@@ -465,7 +469,11 @@ impl FromStr for Network {
     fn from_str(s: &str) -> Result<Self, Self::Err> {
         Ok(match s.to_lowercase().trim() {
             // "mainnet" => Self::Mainnet,
-            "testnet" => Self::Testnet,
+            "testnet" => {
+                return Err(CliError::CommandArgumentError(format!(
+                    "To use testnet, run `movement init --skip-faucet`, follow the prompts to create an account address, then visit the faucet UI  at https://faucet.movementlabs.xyz to fund the account."
+                )));
+            },
```

Comment:
Yes, it should be fine. 

`Network::from_str()` isn't called anywhere else in the crate.

Reply:
Do you see somewhere that it would cause issues @mzabaluev ? It doesn't seem to me that `Network::from_str()` is called anywhere and therefore it won't be a problem.

Reply:
It may be a bit difficult to look into all uses of `FromStr` because it is also used by the `parse` method on string slices in a context where a `Network` value needs to be returned. And that in turn is used under the hood in many places such as `clap` option API.

Reply:
I'd rather look into all places in the CLI code where a `Network` argument is produced and see if "testnet" would make sense there.

Reply:
For now for the sake of getting this approved since I don't have much time to work on the CLI, I've made a change so instead of forbidding "testnet" in network selection, the instruction to use `--skip-faucet` is given during the choose network prompt.